### ISSUE TITLE
Tensorflow guide - trouble on SOFTMAX

### ISSUE THREAD

**Issue by pulliver** [googlecolab/colabtools#1732]
Bug report for Colab: http://colab.research.google.com/.
https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/guide/tensor.ipynb

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior: tensorflow.nn.softmax give a different and wrong result from the softmax algo:
import tensorflow as tf
import numpy as np
c = tf.constant([[4.0, 5.0], [10.0, 1.0]])

#    Compute the softmax
# softmax algo
print (tf.exp(c) / tf.reduce_sum(tf.exp(c)))

# softmax operator
print(tf.nn.softmax(c))

- Describe the expected behavior:
tf.Tensor(             #right
[[2.4558145e-03 6.6755963e-03]
 [9.9074626e-01 1.2226781e-04]], shape=(2, 2), dtype=float32)
tf.Tensor(             #wrong
[[2.6894143e-01 7.3105860e-01]
 [9.9987662e-01 1.2339458e-04]], shape=(2, 2), dtype=float32)
- The web browser you are using (Chrome, Firefox, Safari, etc.):

- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):

**Comment by craigcitro** [comment]
You should report this on the repo where you're loading that notebook (https://github.com/tensorflow/docs/).
